src/development/coding-peer-review.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: article
+title: "Peer review"
+description: "The process for reviewing changes in production code"
+tags: dev
+order: 8
+---
+Here we define the process for reviewing changes in production code using Git and Gitlab.
+
+> __NB: This is a prescriptive process, and should not be deviated from without agreement from your Delivery Manager and Professional Lead__
+
+The high level process is:
+
+![Code review process showing swimlanes for author and reviewer](/development/images/code-review-process.png)
+
+## 1. Author creates Branch
+
+All new code should be committed on a branch from the mainline production code (main).
+
+Development branches should follow the [naming convention](/development/coding-naming-conventions):
+
+> `{change-type}/{JIRA ticket number}-{dash separated human description}`
+
+E.g.
+
+> `feature/LISD-1782-private-pension-order-fix`
+
+## 2. Author commits new code and pushes branch
+
+All changes for the ticket should be committed against the development branch. It is fine to branch from this branch for experimentation, and to merge locally or in a separate review.
+
+Branches should be pushed to gitlab no less than daily to ensure safe-keeping of the code.
+
+### Guidelines for code changes
```

Comment:
How should code changes be documented?  In the README.md broken down by releases?

Reply:
Our Open Source standard docs specifies a `CHANGELOG.md` file.
But I don't think we have a standard approach yet.
I've raised an issue to cover this, and we can look into it.
https://github.com/nhsbsa/nhsbsa-digital-playbook/issues/29